js/upload/src/UploadProgress.svelte

```diff
@@ -76,19 +76,22 @@
 		{files_with_progress.length > 1 ? "files" : "file"}...</span
 	>
 
-	{#if current_file_upload}
+	{#if current_file_upload || files_with_progress.length > 0}
 		<div class="file">
 			<span>
 				<div class="progress-bar">
 					<progress
 						style="visibility:hidden;height:0;width:0;"
-						value={getProgress(current_file_upload)}
-						max="100">{getProgress(current_file_upload)}</progress
+						value={getProgress(current_file_upload || files_with_progress[0])}
```

Comment:
Nit: we could define a new variable that is `current_file_upload || files_with_progress[0]` to avoid duplication